package.json

```diff
@@ -23,6 +23,7 @@
     "generateUpdatesFilesForAllChannels": true,
     "afterPack": "./linux-sandbox-fix.js",
     "afterSign": "./notarize.js",
+    "buildDependenciesFromSource": true,
```

Comment:
This is also why I added this line to test how gh actions behave if prebuilds are disabled in electron-builder